api/metrics-api-jaxrs/src/main/rest-doc/base.adoc

```diff
@@ -139,6 +139,41 @@ Either or both of them can be omitted. `start` defaults to _now_ minus 8 eight h
 /hawkular/metrics/gauges/paris01.web001.example.com.cpu.usage/data?end=20202020
 ----
 
+=== Limit and Order
+
+The API offers the ability to limit the number of data points returned as well specify the oder of the results.
+Limit and order only work with raw data; if used in conjunction with bucketed results, it will return an error.
+
+Usage and default value details:
+
+* default order is descending
+* no limit imposed if parameter not specified
+* limit of 0 or negative is equivalent to no limit
+* if the user specifies limit, then the order is important because it returns data points up to limit in the specified order
+* start time and end time influence the default order if actual limit is specified
+ ** if only start time is specified then order is ascending
+ ** if only end time is specified then order is descending
+ ** if both are specified then order is descending
+ ** if none are specified then order is descending
+ ** order defaults are overridden if order specified
+* default order is descending if limit is not specified regardless of specified time range
+
```

Comment:
You might want to make it a bit clear what happens if you don't specify a start or endtime.
Will this return all the results? Or only ever a maximum of 8 hours (or whatever the limit is)